modules/safer_mysql/variables.tf

```diff
@@ -0,0 +1,338 @@
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  description = "The project ID to manage the Cloud SQL resources"
+}
+
+variable "name" {
+  description = "The name of the Cloud SQL resources"
+}
+
+// required
+variable "database_version" {
+  description = "The database version to use"
+}
+
+// required
+variable "region" {
+  description = "The region of the Cloud SQL resources"
+  default     = "us-central1"
+}
+
+// required
+variable "vpc_network" {
+  description = "Existing VPC network to which instances are connected. The networks needs to be configured with https://cloud.google.com/vpc/docs/configure-private-services-access."
+}
+
+variable "vpc_peering" {
+  description = "google_service_networking_connection object identifying the peering between the producer network and the project subnet. This is not used, but needed to ensure that elements are created in the proper order"
+}
+
+// Master
+variable "tier" {
+  description = "The tier for the master instance."
+  default     = "db-n1-standard-1"
+}
+
+variable "zone" {
+  description = "The zone for the master instance, it should be something like: `a`, `c`."
+}
+
+variable "activation_policy" {
+  description = "The activation policy for the master instance. Can be either `ALWAYS`, `NEVER` or `ON_DEMAND`."
+  default     = "ALWAYS"
+}
+
+variable "authorized_gae_applications" {
+  description = "The list of authorized App Engine project names"
+  default     = []
+}
+
+variable "disk_autoresize" {
+  description = "Configuration to increase storage size"
+  default     = true
+}
+
+variable "disk_size" {
+  description = "The disk size for the master instance"
+  default     = 10
+}
+
+variable "disk_type" {
+  description = "The disk type for the master instance."
+  default     = "PD_SSD"
+}
+
+variable "pricing_plan" {
+  description = "The pricing plan for the master instance."
+  default     = "PER_USE"
+}
+
+variable "maintenance_window_day" {
+  description = "The day of week (1-7) for the master instance maintenance."
+  default     = 1
+}
+
+variable "maintenance_window_hour" {
+  description = "The hour of day (0-23) maintenance window for the master instance maintenance."
+  default     = 23
+}
+
+variable "maintenance_window_update_track" {
+  description = "The update track of maintenance window for the master instance maintenance. Can be either `canary` or `stable`."
+  default     = "canary"
```

Comment:
Would `stable` not be preferable for a secure/production-ready workload?

Reply:
Changed to stable. What do you think of the default maintenance window? Should we force users to pick one instead?

Reply:
Default is fine.